src/components/FaqAccordian/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react"
+import { MdChevronRight } from "react-icons/md"
+import * as AccordionPrimitive from "@radix-ui/react-accordion"
+
+import { cn } from "@/lib/utils/cn"
+
+import * as RootAccordion from "../../../tailwind/ui/accordion"
+
+const AccordionTrigger = React.forwardRef<
+  React.ElementRef<typeof AccordionPrimitive.Trigger>,
+  React.ComponentPropsWithoutRef<typeof AccordionPrimitive.Trigger>
+>(({ className, children, ...props }, ref) => (
+  // p-4 hover:bg-background-highlight
+  <AccordionPrimitive.Header className="flex w-full">
```

Comment:
Is there a reason we didn't go with the existing `AccordionTrigger` from `tailwind/ui/accordion.tsx`, instead of rebuilding from the Radix-UI primitive?

Reply:
It's used for grouping styles with `MdChevronRight` ==> `"group-hover:border-primary group-hover:text-primary group-hover:shadow-md group-hover:shadow-primary-low-contrast"`